src/main/java/dk/cngroup/lentils/controller/StatisticsController.java

```diff
@@ -0,0 +1,79 @@
+package dk.cngroup.lentils.controller;
+
+import dk.cngroup.lentils.entity.CypherStatus;
+import dk.cngroup.lentils.service.StatisticsService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/game/statistics")
+public class StatisticsController {
+
+    private static final String STATISTICS_VIEW = "statistics/statistics";
+
+    private static final String HINT_TAKEN_COUNT_HEADER = "Seřazeno podle celkového počtu použitých nápověd";
+    private static final String HINT_TAKEN_POINTS_HEADER = "Seřazeno podle celkového počtu bodů za nápovědy";
+    private static final String SKIPPED_TEAMS_COUNT_HEADER = "Seřazeno podle počtu týmů, kteří přeskočili";
+    private static final String SOLVED_TEAMS_COUNT_HEADER = "Seřazeno podle počtu týmů, kteří vyřešili";
+
+    private static final String HINT_TAKEN_COUNT_COLUMN = "Počet nápověd";
+    private static final String HINT_TAKEN_POINTS_COLUMN = "Body za nápovědy";
+    private static final String SKIPPED_TEAMS_COLUMN = "Týmů přeskočilo";
+    private static final String SOLVED_TEAMS_COLUMN = "Týmů vyřešilo";
+
+    private final StatisticsService statisticsService;
+
+    @Autowired
+    public StatisticsController(final StatisticsService statisticsService) {
+        this.statisticsService = statisticsService;
+    }
+
+    @GetMapping
+    public String statisticsView() {
+        return STATISTICS_VIEW;
+    }
+
+    @GetMapping("/cyphersByCountOfHintsTaken")
+    public String countOfHintTakenForCyphers(final Model model) {
+        model.addAttribute("cyphersStatsMap", statisticsService.getNumberOfHintsTakenOfCyphers());
```

Comment:
Jo, ještě bych tě poprosil, jestli by si mohl udělat konstanty pro ty klíče v modelu. Začali jsme to dávat, protože se to často opakuje a líp se to spravuje, když je to několikrát i v šablonách.

např.
private static final String TEMPLATE_ATTR_STATS_MAP = "cyphersStatsMap";
atd.

Reply:
dodělám ;) díky